packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -84,15 +78,6 @@ const getStateStyles = (
       case 'info':
         return [
           activeStyle,
-          isDisabled &&
```

Comment:
This styles were never used. If the `isDisabled` prop is true, we return other styles previously in this function.